modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -97,6 +100,8 @@
 
     private ExperimentTagAccessor experimentTagAccessor;
 
+    private PrioritiesAccessor prioritiesAccessor;
```

Comment:
@bargenilesh Although I don't see the getters and setters of the other member fields being used anywhere except in tests, but just for consistency, would suggest adding getters and setters for this field as well. We might need to do a complete cleanup later if we do not expect these fields to be mutated once set in the constructor using Guice. We might want to then declare these fields `final` as well.